metron-platform/metron-solr/metron-solr-common/src/main/java/org/apache/metron/solr/writer/SolrClientFactory.java

```diff
@@ -31,21 +31,33 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
-public class MetronSolrClient extends CloudSolrClient {
+public class SolrClientFactory {
 
   private static final Logger LOG = LoggerFactory
-          .getLogger(MetronSolrClient.class);
+          .getLogger(SolrClientFactory.class);
 
 
-  public MetronSolrClient(String zkHost) {
-    super(zkHost);
+  public static CloudSolrClient create(String zkHost) {
+    CloudSolrClient.Builder builder = getBuilder(zkHost);
+    return builder.build();
   }
 
-  public MetronSolrClient(String zkHost, Map<String, Object> solrHttpConfig) {
-    super(zkHost, HttpClientUtil.createClient(toSolrProps(solrHttpConfig)));
+  public static CloudSolrClient create(String zkHost, Map<String, Object> solrHttpConfig) {
```

Comment:
Have we always used the `CloudSolrClient` class rather than `SolrClient`?  Just want to double check.

Reply:
Yes I believe so.  ‘MetronSolrClient’ wrapped a ‘SolrCloudClient’ instance.